src/bootstrap/bootstrap.py

```diff
@@ -390,5 +398,9 @@ def main():
     env["BOOTSTRAP_PARENT_ID"] = str(os.getpid())
     rb.run(args, env)
 
+    end_time = time()
+
+    print("Build completed in %s" % format_build_time(end_time - start_time))
```

Comment:
Nit: No need for a format string here, just use `print("Build completed in in" + format_build_time(end_time - start_time))`

Aside: Why doesn't this script have `from __future__ import print_function`? It would make printing simpler since one wouldn't have to perform manual string interpolation/concatenation everywhere).